crates/consensus-logic/src/csm/worker.rs

```diff
@@ -153,7 +153,7 @@ fn process_msg<D: Database>(
                 if ev_idx < *idx {
                     warn!(%ev_idx, "Applying missed sync event.");
                 }
-                handle_sync_event_with_retry(state, engine, ev_idx, status_rx, shutdown)?;
+                handle_sync_event_with_retry(state, engine, ev_idx, status_tx, shutdown)?;
```

Comment:
This diff seems wrong no?
I expect that you would never replace a `foo_rx` for a `foo_tx` no?

Reply:
That's a valid concern. Actually, the function argument was wrongly named, this change uses the correct name. :D